src/devices/Tca9548a/README.md

```diff
@@ -0,0 +1,49 @@
+# TCA9548A - TCA9548A Low-Voltage 8-Channel I2C Switch with Reset
+
+The TCA9548A device has eight bidirectional translating switches that can be controlled through the I2C bus. The SCL/SDA upstream pair fans out to eight downstream pairs, or channels.
+
+All operations for basic usage has been developed. If needed, RESET operation can also be implemented.
+
+## Documentation
+
+![TCA9548A](./TCA9548A.jpg)
+
+TCA9548A [datasheet](https://www.ti.com/lit/ds/symlink/tca9548a.pdf)
+
+You will find this device as ["Adafruit TCA9548A 1-to-8 I2C Multiplexer Breakout"](https://learn.adafruit.com/adafruit-tca9548a-1-to-8-i2c-multiplexer-breakout)
+
+## Usage
+
+Create a ```Tca9548A``` class and pass the I2C device. The default I2C address is provided in the class.
```

Comment:
Your example is a bit pointless and doesn't really show an useful use case. Can you extend the sample (and the diagram) to include for instance to BMEXXX sensors with the same address?